packages/libs/react-components/src/components/Input/Input.tsx

```diff
@@ -16,8 +16,8 @@ export interface IInputProps
   > {
   as?: 'input';
   leadingText?: string;
-  leftPanel?: typeof SystemIcons[keyof typeof SystemIcons];
-  rightPanel?: typeof SystemIcons[keyof typeof SystemIcons];
+  leftPanel?: (typeof SystemIcons)[keyof typeof SystemIcons];
+  rightPanel?: (typeof SystemIcons)[keyof typeof SystemIcons];
```

Comment:
Were you able to figure out what was applying this? I've seen other PRs auto-formatting this back

Reply:
I think I've finally found it! Tried removing my global install of prettier and the change gets reverted upon committing the affected files 🥳🥳 will revert these in the currently open PRs that also got affected by t